src/utils/dmg.c

```diff
@@ -904,6 +979,14 @@ query obj layout options: \n\
   --pool=UUID	pool uuid\n\
   --cont=UUID	container uuid\n\
   --oid=oid	object oid.\n");
+	printf("\
```

Comment:
(style)  Avoid unnecessary line continuations
(style)  Avoid line continuations in quoted strings